spec/factories/info_requests.rb

```diff
@@ -6,34 +6,34 @@
     public_body
     user
 
-    after_create do |info_request, evaluator|
-      FactoryGirl.create(:initial_request, :info_request => info_request)
+    after(:create) do |info_request, evaluator|
+      create(:initial_request, :info_request_id => info_request.id)
```

Comment:
Out of interest, why did `:info_request` need to get changed to `:info_request_id` here?